ui/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Is the repo level .gitignore not working?

Reply:
Nothing against module specific ignores, given how broad `/build` is I was wondering if we had any tools issue that required it in lower levels, noticed it previously in `app` and was curious.